collections-csharp/ConcurrentQueueInCSharp/ConcurrentQueueInCSharp.Console/OrderMessageBus.cs

```diff
@@ -0,0 +1,30 @@
+﻿using ConcurrentQueueInCSharp.Models;
+using System.Collections.Concurrent;
+
+namespace ConcurrentQueueInCSharp
+{
+    public class OrderMessageBus
+    {
+        private readonly ConcurrentQueue<Order> _queue = new ConcurrentQueue<Order>();
+
+        public int Count
+        {
+            get
+            {
+                return _queue.Count;
+            }
+        }
+
+        public void Add(Order order)
+        {
+            if (order == null) throw new ArgumentNullException(nameof(order));
```

Comment:
We can clean this up with the `ArgumentNullException.ThrowIfNull(order);`. The throw helper uses the `CallerArgumentExpression` attribute to pull the argument name.

Reply:
Nice one. I didn't know it.